cmd/controller/main.go

```diff
@@ -112,15 +136,19 @@ func main() {
 	fasController := fleetautoscalers.NewController(wh, health,
 		kubeClient, extClient, agonesClient, agonesInformerFactory)
 
+	rs := []runner{
+		wh, gsController, gsSetController, fleetController, faController, fasController, metricsController, server,
+	}
+
 	stop := signals.NewStopChannel()
 
 	kubeInformationFactory.Start(stop)
 	agonesInformerFactory.Start(stop)
 
-	rs := []runner{
-		wh, gsController, gsSetController, fleetController, faController, fasController, healthServer{handler: health},
-	}
 	for _, r := range rs {
+		if r == nil {
```

Comment:
Curious - what is this for?

Reply:
Metrics controller can be nil, if it's switche off